charts/opentelemetry-demo/values.yaml

```diff
@@ -614,7 +614,7 @@ components:
             value: http://$(OTEL_COLLECTOR_NAME):4318
         resources:
           limits:
-            memory: 75Mi
+            memory: 160Mi
```

Comment:
This should be 100Mi to match what is in the Demo as per this [PR](https://github.com/open-telemetry/opentelemetry-demo/pull/2120)

```suggestion
            memory: 100Mi
```

Reply:
@puckpuck I was still getting OOMs in k8s at 100Mi

Reply:
<img width="948" alt="image" src="https://github.com/user-attachments/assets/7c5a3984-7426-41c1-9db1-727459f28d5b" />

Reply:
100Mi to 160Mi might have been more a jump than necessary -- I just didn't want to have to keep dealing with it at that moment. 😄 

It seems stable for me at 120Mi as well after more testing. Does this value sound reasonable?

Reply:
I'm interested to know what kind of hardware you are running the Demo on to get OOM still? Under normal conditions flagd-ui should consume less than 60M

Also this field needs to be the same value as what we have in the Demo repo so we don't drift away in configuration. Right now the Demo repo has this value at 100M

Reply:
Let's set it to 100M and I can update my own config to increase it locally. I'm using an arm Mac (m3 max).